include/pstore/exchange/export_fixups.hpp

```diff
@@ -46,8 +49,11 @@ namespace pstore {
                                              [&] (OStream & os1, repo::external_fixup const & xfx) {
                                                  os1 << R"({"name":)" << names.index (xfx.name)
                                                      << R"(,"type":)"
-                                                     << static_cast<unsigned> (xfx.type)
-                                                     << R"(,"offset":)" << xfx.offset
+                                                     << static_cast<unsigned> (xfx.type);
+                                                 if (xfx.is_weak) {
+                                                     os1 << R"(,"is_weak":)" << xfx.is_weak;
+                                                 }
```

Comment:
Because the true case is unusual. By having a default of 'false' and allowing the field to be omitted, we reduce the output file size and reduce the amount of work that the parser must do on input.